### PR TITLE
List "view/edit all" groups in group permission field

### PR DESCRIPTION
**Pull request by chillu**
See https://github.com/silverstripe/silverstripe-cms/issues/794

Similar PR for siteconfig: https://github.com/silverstripe/silverstripe-siteconfig/pull/59

### CODE REVIEW

**Comment by tractorcow** [comment]
@chillu screenshot would help review ;)

**Comment by tractorcow** [comment]
image

Looks good.

File: code/Model/SiteTree.php [hunk 1]
```diff
@@ -2017,6 +2023,22 @@ public function getSettingsFields()
         unset($editorsOptionsSource[InheritedPermissions::ANYONE]);
         $editorsOptionsField->setSource($editorsOptionsSource);
 
+        if ($viewAllGroupsMap) {
+            $viewerGroupsField->setDescription(_t(
+                'SilverStripe\\CMS\\Model\\SiteTree.ViewerGroupsFieldDescription',
```
Comment (tractorcow) [inline code/Model/SiteTree.php]: Please use all-caps for right hand side of localisation keys (convention).

Comment (chillu) [inline code/Model/SiteTree.php]: Done

**Comment by tractorcow** [comment]
Restarted behat tests... some weird behaviour.

**Pull request merged**